ocamldoc/odoc_html.ml

```diff
@@ -2618,15 +2618,17 @@ class html =
       try
         let chanout = open_out (Filename.concat !Global.target_dir self#index) in
         let b = new_buf () in
-        let title = match !Global.title with None -> "" | Some t -> self#escape t in
         bs b doctype ;
         bs b "<html>\n";
         self#print_header b self#title;
         bs b "<body>\n";
 
-        bs b "<h1>";
-        bs b title;
-        bs b "</h1>\n" ;
+        let () = match !Global.title with
+          | None -> ()
+          | Some t ->
+              let () = bs b "<h1>" in
+              let () = bs b (self#escape t) in
+              bs b "</h1>\n" in
```

Comment:
Agreed and corrected but this one was intentional. `;` precedence is a nightmare for me and after having been too many time by
```
if something then foo;
bar
```
that even tuareg wrongly indent. I decided to ban `;` :-) (I don't find `let () = ... in` less odd than the `(match ...); ...` I had to make here. Anyway, taste and colours, sorry to not respect the format of the fil I edited...)